extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -248,11 +257,84 @@ private JibContainerBuilder createContainerBuilderFromFastJar(JibConfig jibConfi
             entrypoint.add(JarResultBuildStep.QUARKUS_RUN_JAR);
         }
 
+        List<AppArtifact> fastChangingLibs = new ArrayList<>();
+        List<AppDependency> userDependencies = curateOutcome.getEffectiveModel().getUserDependencies();
+        for (AppDependency dep : userDependencies) {
+            AppArtifact artifact = dep.getArtifact();
+            if (artifact == null) {
+                continue;
+            }
+            String artifactVersion = artifact.getVersion();
+            if ((artifactVersion == null) || artifactVersion.isEmpty()) {
+                continue;
+            }
+            if (artifactVersion.toLowerCase().contains("snapshot")) {
```

Comment:
Maven supports adding timestamps to the version. Such artifacts also qualify as fast changing.
Not sure if anyone is really using this feature. Also the timestamp has a configurable format, which makes it hard to detect with a regex. But I am just mentioning it ...

Reply:
That's great info!

I do plan to improve this feature with project level dependency detection and what you mention is also another good candidate for improvement

Reply:
FWIW, Maven also checks for LATEST (but not for such timestamped versions, at least not specifically):
https://github.com/apache/maven/blob/master/maven-artifact/src/main/java/org/apache/maven/artifact/DefaultArtifact.java#L519-L523

Reply:
> Not sure if anyone is really using this feature.

Can be really useful e.g. when deploying snapshots to sonatype so that issue reporters can try a fix without building a project themselves.
E.g.: https://github.com/vackosar/gitflow-incremental-builder/issues/241#issuecomment-684075071

Reply:
Thanks @famod !